src/components/forms/AppTextArea.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <label v-if="label" class="block mb-1.5 font-semibold" :for="inputType"
+    >{{ formattedLabel }}
+  </label>
+  <textarea
+    :id="inputType"
+    class="
+      p-2
+      w-full
+      border border-primary-40
+      rounded
+      focus:outline-none focus:shadow-input
+    "
+    :type="inputType"
```

Comment:
I don't think we need a type attribute for this component as it's not relevant to a `<textarea>`

Reply:
Done: https://github.com/beabee-communityrm/beabee-frontend/pull/104/commits/578f343a4e4679ed35823e36a989aa613b9789b8